crates/solidity/testing/utils/src/cst_snapshots/test_nodes.rs

```diff
@@ -12,6 +19,7 @@ pub enum TestNodeKind {
     Rule(RuleKind),
     Token(TokenKind),
     Trivia(TokenKind),
+    Error,
```

Comment:
I suggest storing/rendering the error message from `ErrorNode` as well, instead of dropping it.

Reply:
I understand the consensus is to drop the Node, so this probably won't end up in a CST